app/src/main/res/values/strings.xml

```diff
@@ -673,41 +673,41 @@
     <string name="dsa_affine_y">Affine y coordinate</string>
     <string name="critical_exts">Critical Extensions</string>
     <string name="non_critical_exts">Non-critical Extensions</string>
-    <string name="usage_access_not_supported">Could not request usage access as the corresponding settings page does not exist.</string>
+    <string name="usage_access_not_supported">Could not request usage access, as the corresponding settings page does not exist.</string>
```

Comment:
It's incorrect to add a comma before _as_.

Reply:
It is correct here though https://magnumproofreading.com/blog/proper-punctuation/comma-before-as/